viabel/objectives.py

```diff
@@ -174,7 +183,7 @@ def RGE(var_param):
             epsilon_sample = (z_samples - m_mean) / s_scale
             # elbo = np.mean(self._model(z_samples) - approx.log_density(var_param, z_samples))
             if self._use_path_deriv:
-                var_param_stopped = getval(var_param)
+                var_param_stopped = device_get(var_param)
```

Comment:
why do you use `var_param.primal` above but not here?

Reply:
This `var_param` is Jax.numpy.array, and I use `device_get` to obtain the array, while all other `var_param` are Jax.Tracer.